CHANGELOG.md

```diff
@@ -112,40 +125,50 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * The `diff.stat()` template method now provides methods to get summary values.
 
-* New `git.sign-on-push` config option to automatically sign commits which are being
-  pushed to a Git remote.
+* Add templater support for rendering commit signatures.
```

Comment:
Is this the same as the bullet you added under "Release highlights"? I find that version easier to read. Can we use that here too (instead of the current text)? Or maybe this is redundant with the next bullet so we should drop this one?

Reply:
Copied the highlight line and merged two "new features" items into one.